Resources/Prototypes/DeltaV/Recipes/Construction/Graphs/structures/disco_tile.yml

```diff
@@ -0,0 +1,57 @@
+- type: constructionGraph
+  id: DiscoFloor
+  start: start
+  graph:
+    - node: start
+      actions:
+        - !type:DestroyEntity {}
+      edges:
+        - to: discoSquare
+          steps:
+            - material: Plasteel
+              amount: 5
+            - material: Glass
+              amount: 6
+            - material: Cable
+              amount: 5
+              doAfter: 8
+            - material: Gold
+              amount: 4
+              doAfter: 5
+
+        - to: discoPart
+          steps:
+            - material: Plasteel
+              amount: 5
+            - material: Glass
+              amount: 6
+            - material: Cable
+              amount: 5
+              doAfter: 8
+            - material: Gold
+              amount: 4
+              doAfter: 5
+
+    - node: discoSquare
+      entity: DiscoFloor
+      edges:
+        - to: start
+          completed:
+            - !type:SpawnPrototype
+              prototype: SheetSteel5
+              amount: 5
+          steps:
+            - tool: Prying
+              doAfter: 5
+
+    - node: discoPart
+      entity: DiscoFloor
+      edges:
+        - to: start
+          completed:
+            - !type:SpawnPrototype
+              prototype: SheetSteel5
+              amount: 5
+          steps:
+            - tool: Prying
+              doAfter: 5
```

Comment:
why have 2 of the same graph

Reply:
Guh, I just realized how to get it to work without that. fixed!